src/physics/em/detail/SBPositronXsCorrector.hh

```diff
@@ -0,0 +1,68 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file SBPositronXsCorrector.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Types.hh"
+#include "physics/base/Units.hh"
+#include "physics/material/ElementView.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Scale SB differential cross sections for positrons.
+ *
+ * This correction factor appears in the bowels of \c
+ * G4SeltzerBergerModel::SampleEnergyTransfer in Geant4 and scales the SB cross
+ * section by a factor
+ * \f[
+   \frac{\sigma_\mathrm{corrected}}{\sigma}
+       = \alpha Z [ \beta(E) - \beta(E_c) ]
```

Comment:
```suggestion
       = \exp(\alpha Z [ \beta(E) - \beta(E_c) ])
```
For reference, it looks like this factor comes from [this paper](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.33.3002)

Reply:
Thanks for the correction and reference!